options.html

```diff
@@ -8,40 +8,55 @@
 </head>
 
 <body id="options-body">
-  <div>
-    <label for="show_notifications"
-           title="Show a notification whenever an item is uploaded to the RECAP Archive on CourtListener.com">
-      <div class="item">
-        <span class="icon">
-          <input id="show_notifications" type="checkbox">
-        </span>
-        Notify me when RECAP uploads a file to the Archive
-      </div>
-    </label>
-    <label for="recap_link_popups"
-      title="Pop up a dialog whenever you click a RECAP link to a free document">
-      <div class="item">
-        <span class="icon">
-          <input id="recap_link_popups" type="checkbox">
-        </span>
-        Confirm before opening free documents
-      </div>
-    </label>
-    <fieldset>
-      <legend>PDF Filenames</legend>
-      <div>
-        <input type="radio" id="ia_style_filenames"
-               name="filename_style" value="ia_style_filenames">
-        <label for="ia_style_filenames">Use Internet Archive style (e.g.
-          gov.uscourts.cand.201881.46.0.pdf)</label>
-      </div>
-      <div>
-        <input type="radio" id="lawyer_style_filenames"
-               name="filename_style" value="lawyer_style_filenames">
-        <label for="lawyer_style_filenames">Use Lawyer style (e.g.
-          N.D.Cal._3&#8209;08&#8209;cv&#8209;03251_46_0.pdf)</label>
-      </div>
-    </fieldset>
+  <fieldset><legend>Notifications and popups</legend>
+    <div>
+      <label for="show_notifications"
+             title="Show a notification whenever an item is uploaded to the RECAP Archive on CourtListener.com">
+	<div class="item">
+          <span class="icon">
+            <input id="show_notifications" type="checkbox">
+          </span>
+          Notify me when RECAP uploads a file to the Archive
+	</div>
+      </label>
+      <label for="recap_link_popups"
+	     title="Pop up a dialog whenever you click a RECAP link to a free document">
+	<div class="item">
+          <span class="icon">
+            <input id="recap_link_popups" type="checkbox">
+          </span>
+          Confirm before opening free documents
+	</div>
+      </label>
+  </fieldset>
+  
+  <fieldset><legend>PDF Filenames</legend>
+    <div>
+      <input type="radio" id="ia_style_filenames"
+             name="filename_style" value="ia_style_filenames">
+      <label for="ia_style_filenames">Use Internet Archive style (e.g.
+        gov.uscourts.cand.201881.46.0.pdf)</label>
+    </div>
+    <div>
+      <input type="radio" id="lawyer_style_filenames"
+             name="filename_style" value="lawyer_style_filenames">
+      <label for="lawyer_style_filenames">Use Lawyer style (e.g.
+        N.D.Cal._3&#8209;08&#8209;cv&#8209;03251_46_0.pdf)</label>
+    </div>
+  </fieldset>
+  
+  <!-- TBD: Optimize PACER -->
```

Comment:
Can you put a reference to the issue number in the comment here, or remove it completely? I don't know when we'll get to that ticket, so I want to make sure we don't have vestigial bits of it in here that we might not understand when we get around to it.